### DISCUSSION TITLE
How to `purge` but for target?

### DISCUSSION THREAD

**Discussion by latipun7** [twpayne/chezmoi#1813] [Q&A] [answered]
Current behavior of `chezmoi purge` based on docs is 
> _"Remove chezmoi’s configuration, state, and source directory, but leave the target state intact."_

How to do the same thing, but for target? I want conveniently to remove chezmoi’s configuration, cache, state, and target, but **leave the source directory** intact (option to remove source directory is cool).

I'd like to go back to original state before running `chezmoi apply` for the first time or `chezmoi init <user> --apply` (before having chezmoi and dotfiles), so I could re-evaluate the idempotency.

Thank you.

**Comment by twpayne** [comment, marked as answer]
You can remove all chezmoi-managed files in the target state with:
```console
$ rm -i $(chezmoi managed)
```

For the other directories you mention, you can use `rm`.

**Reply by latipun7** [reply]
> ```
> $ rm -i $(chezmoi managed)
> ```

~~I have trouble doing that, so I ask this question.~~ SOLVED

**Reply by twpayne** [reply]
Sorry I do not have time to teach you how to use a shell.

`chezmoi managed` will give you a list of targets managed by chezmoi. You can then use `rm` to delete them.

**Reply by latipun7** [reply]
I'm sorry too for the trouble. I already solved the issue, yeah, it's just me for the little knowledge of shell. Thank you 🎉
